src/console_implementation/drawable_player.cpp

```diff
@@ -1,7 +1,7 @@
 #include "drawable_player.hpp"
 #include "../business_logic/include/Tposition.hpp"
```

Comment:
Thanks to CMake integration there is no need for relative paths.
```suggestion
#include "Tposition.hpp"
```